[yangster-docker@next, theia-full-next] not working

It builds but the app can't start

Client-side error:
```
index.js:95 Error: Cannot apply @injectable decorator multiple times.
    at injectable.js:8
    at Reflect.js:540
    at Object.<anonymous> (Reflect.js:130)
    at a (decorator_utils.js:42)
    at Object.t.decorate (decorator_utils.js:55)
    at Object.<anonymous> (widget.ts:25)
    at n (bootstrap:19)
    at Object.<anonymous> (index.ts:17)
    at n (bootstrap:19)
    at Object.<anonymous> (dialogs.ts:20)
```
This is caused by the fact that the app pulls 2 versions of `@theia/core`. The root cause looks to be the `theia-yang-extension@next` extension:

https://github.com/theia-ide/yangster/issues/85

--- a/.travis.yml
+++ b/.travis.yml
@@ -26,8 +26,6 @@ env:
   - NPM_TAG=next IMAGE_NAME=theia-java NODE_VERSION=10
   - NPM_TAG=latest IMAGE_NAME=theia-go NODE_VERSION=10
   - NPM_TAG=next IMAGE_NAME=theia-go NODE_VERSION=10
-  - NPM_TAG=latest IMAGE_NAME=yangster NODE_VERSION=10
-  - NPM_TAG=next IMAGE_NAME=yangster NODE_VERSION=10
   - NPM_TAG=latest IMAGE_NAME=theia-python NODE_VERSION=10
   - NPM_TAG=next IMAGE_NAME=theia-python NODE_VERSION=10
   - NPM_TAG=latest IMAGE_NAME=theia-swift NODE_VERSION=10

--- a/theia-full-docker/next.package.json
+++ b/theia-full-docker/next.package.json
@@ -58,7 +58,6 @@
         "@theia/userstorage": "next",
         "@theia/variable-resolver": "next",
         "@theia/workspace": "next",
-        "theia-yang-extension": "next",
         "typescript": "latest"
     },
     "resolutions": {

